src/tree/images/ImageTreeItem.ts

```diff
@@ -59,25 +57,11 @@ export class ImageTreeItem extends AzExtTreeItem {
     }
 
     public async getImage(): Promise<Image> {
-        return callDockerode(() => ext.dockerode.getImage(this.imageId));
+        return callDockerode(() => ext.dockerode.getImage(this.fullTag));
```

Comment:
Removal fails here for images that are tagged `<none>`, i.e. those where a newer version of the same image has been downloaded.